editoast/src/views/rolling_stocks/mod.rs

```diff
@@ -74,36 +76,53 @@ async fn get(db_pool: Data<DbPool>, path: Path<i64>) -> Result<Json<RollingStock
 
 #[derive(Debug, Clone, Deserialize, Serialize)]
 pub struct RollingStockForm {
-    #[serde(flatten)]
-    pub common: RollingStockCommon,
```

Comment:
Why not remove the `railjson_version` version field from `RollingStockCommon` instead ?

Reply:
Done